server/tests/utils/server/servers.ts

```diff
@@ -35,7 +35,7 @@ interface ServerInfo {
   }
 }
 
-function flushAndRunMultipleServers (totalServers) {
+function flushAndRunMultipleServers (totalServers, configOverrideFn = (numServer: number) => undefined) {
```

Comment:
please remove configOverrideFn

Reply:
How do I disable transcoding then for the first server? (if I should)